ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-isis.j2

```diff
@@ -36,6 +36,29 @@
 {%     if router_isis.segment_routing_mpls.enabled is arista.avd.defined %}
 | SR MPLS Enabled | {{ router_isis.segment_routing_mpls.enabled }} |
 {%     endif %}
+{%     if router_isis.redistribute_routes is arista.avd.defined %}
+
+### ISIS Route Redistribution
+
+| Route Type | Route-Map | Include Leaked |
+| ---------- | --------- | -------------- |
+{%         for redistribute_route in router_isis.redistribute_routes | arista.avd.convert_dicts('source_protocol') | arista.avd.natural_sort('source_protocol') %}
+{%             set src_protocol = redistribute_route.source_protocol | arista.avd.default('-') %}
+{%             set route_map = redistribute_route.route_map | arista.avd.default('-') %}
+{%             if redistribute_route.source_protocol is arista.avd.defined('static') or redistribute_route.source_protocol is arista.avd.defined('connected') or redistribute_route.source_protocol is arista.avd.defined('ospf') %}
+{%                 set include_leaked = redistribute_route.include_leaked | arista.avd.default('-') %}
+{%             else %}
+{%                 set include_leaked = include_leaked | arista.avd.default('-') %}
```

Comment:
there is no `include_leaked` set anywhere, so this will always be `-`. You could just `set include_leaked = '-'`